maker/src/connection.rs

```diff
@@ -481,7 +490,18 @@ impl Actor {
                 }
             },
         );
-        tasks.add(this.send_interval(self.heartbeat_interval, move || SendHeartbeat(identity)));
+
+        let daemon_semver = parse_vergen_version(&daemon_version);
+        let no_need_for_heartbeats =
+            semver::VersionReq::parse(">= 0.4.20").expect("to parse VersionReq");
```

Comment:
I think this should be `0.4.21`, i.e. only from the next release.

Reply:
indeed, I will change this to `0.4.21` and downgrade the tracing warning on the taker side to trace (as this change will invariably still send heartbeats to the taker right until the next release time).